.circleci/config.yml

```diff
@@ -222,26 +226,51 @@ commands:
 jobs:
   api-build-test:
     <<: *java_defaults
+    parallelism: 3
     steps:
       - checkout_init_git
       - ensure_branch_has_changes:
           dir_name: "api"
       - manage_api_cache:
           restore: true
       - run:
-          name: Validate Swagger definitions
+          name: Run Java unit tests
           working_directory: ~/workbench/api
-          command: ./project.rb validate-swagger --project-prop verboseTestLogging=yes
+          # Refers to https://circleci.com/docs/2.0/language-java/#sample-configuration
+          command: |
+            cd ~/workbench/api/src/test/java/org/pmiops/workbench/api
```

Comment:
This is not correct. `api` is one subdirectory of many that contain unit tests

I would change this to `pushd src/test`, then `popd` later